### DISCUSSION TITLE
After setting INSTALLED_APPS getting No module named 'fobi'

### DISCUSSION THREAD

**Discussion by michaeloliveraz** [barseghyanartur/django-fobi#289] [Q&A] [answered]
pip install django-fobi worked fine.

My settings.py INSTLLED_APPS

`'rest_framework_swagger', # optional, creates browsable API docs
     # `django-fobi` core
    'fobi',

    # `django-fobi` themes
    'fobi.contrib.themes.bootstrap3', # Bootstrap 3 theme
    'fobi.contrib.themes.foundation5', # Foundation 5 theme
    `
Any idea what to change?

**Comment by AliVleotsky** [comment, marked as answer]
I had moved the project from one computer to another and venv was not resolving apps and modules.  Recreated venv and it started working again
